docs/builders/vsphere-supervisor.mdx

```diff
@@ -0,0 +1,204 @@
+---
+modeline: |
+  vim: set ft=pandoc:
+description: >
+  The Packer builder that deploys a VM-Service vm to a vSphere Supervisor cluster.
+page_title: vSphere Supervisor - Builders
+sidebar_title: Supervisor
+---
+
+# VMware vSphere Supervisor Builder
+
+Type: `vsphere-supervisor`
+Artifact BuilderId: `vsphere.supervisor`
+
+This builder deploys new VMs to a vSphere Supervisor cluster.
+
+- It uses kubeconfig file to connect to the vSphere Supervisor cluster.
+- It uses the VM-Service API to deploy and configure the source VM.
+- It uses the Packer provisioners to customize the VM after deployment.
+- Planned enhancements that will introduce the ability to publish the customized VM as a new VM image to the vSphere endpoint.
+
+## Examples
+
+Example Packer template:
+
+<Tabs>
+<Tab heading="HCL2">
+
+```hcl
+source "vsphere-supervisor" "example-vm" {
+  image_name = "<Image name of the source VM, e.g. 'ubuntu-impish-21.10-cloudimg'>"
+  class_name = "<VM class that describes the virtual hardware settings, e.g. 'best-effort-large'>"
+  storage_class = "<Storage class that provides the backing storage for volume, e.g. 'wcplocal-storage-profile'>"
+}
+
+build {
+  sources = ["source.vsphere-supervisor.example-vm"]
+}
+```
+
+</Tab>
+<Tab heading="JSON">
+
+```json
+{
+  "builders": [
+    {
+      "type": "vsphere-supervisor",
+      "image_name": "<Image name of the source VM, e.g. 'ubuntu-impish-21.10-cloudimg'>"
+      "class_name": "<VM class that describes the virtual hardware settings, e.g. 'best-effort-large'>",
+      "storage_class": "<Storage class that provides the backing storage for volume, e.g. 'wcplocal-storage-profile'>"
+    }
+  ]
+}
+```
+
+</Tab>
+</Tabs>
+
+Please check the [examples/supervisor folder](https://github.com/hashicorp/packer-plugin-vsphere/tree/main/builder/vsphere/examples/supervisor) for more complete examples.
```

Comment:
Just calling out that the examples are available via GitHub and not within the docs. 
```suggestion
Please check the [examples/supervisor folder](https://github.com/hashicorp/packer-plugin-vsphere/tree/main/builder/vsphere/examples/supervisor) within the GitHub repository for more complete examples.
```